compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaObjectFieldSymbol.java

```diff
@@ -86,14 +87,30 @@ public List<AnnotationSymbol> annotations() {
 
         List<AnnotationSymbol> annots = new ArrayList<>();
         SymbolFactory symbolFactory = SymbolFactory.getInstance(this.context);
-        for (AnnotationAttachmentSymbol annot : bField.symbol.getAnnotations()) {
+        for (org.ballerinalang.model.symbols.AnnotationAttachmentSymbol annot : bField.symbol.getAnnotations()) {
```

Comment:
Any possibility of getting `CCE` here ?

Reply:
No, here `getAnnotations()` returns symbols of `org.ballerinalang.model.symbols.AnnotationAttachmentSymbol`